pyproject.toml

```diff
@@ -29,10 +29,3 @@ urls = { Homepage = "https://github.com/tpgillam/mt2" }
 requires = ["setuptools>=61.0", "numpy"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools.packages.find]
-where = ["src"]
-include = ["mt2*"]
-namespaces = false
-
-[tool.setuptools.package-data]
-mt2 = ["src/_mt2/*.h", "src/_mt2/*.cpp"]
```

Comment:
This was my earlier attempt to avoid having a MANIFEST.in; actually this doesn't seem to have any effect, so we can get rid of it.